### ISSUE TITLE
Staggered orders missing orders

### ISSUE THREAD

**Issue by AndreiMihalea** [Drakkar-Software/OctoBot#1333]
## Environment description:
  * OS: Ubuntu 20.04
  * Python Version: 3.7.0
  * Last Commit ID: 0d65a1bc1712823684aa0532d4b9177cd2e9bc17
  * Branch: Dev
 
## Problem description
Using the Binance trading platform (only one I've tried with), some orders are not being created, even though the bot says they are. They appear in logs for as long as the bot runs, but as soon as it gets restarted the number of orders will become the real one, shown on Binance.

**Expected behavior**
When an order is completed, another order should be created.

### Steps to reproduce:

  1. Enable the real trader
  2. Configure the staggered orders strategy
  3. Let the bot run for some time (sometimes it takes hours until the first order goes missing)
  
### Observed Results:

  * What happened? The number of orders on Binance starts to decrease, while the number of orders stays the same in the bot's logs. When the bot is restarted, the orders shown are the ones that remained on Binance.
  * What did you expect to happen? The number of orders on Binance and the ones shown by the bot should be the same.

### Relevant code exceptions or logs:
 
Here, I have a fragment of a logfile before closing the bot, where we can see there are 147 open orders (on Binance, there were only 143)
```
2020-10-02 15:03:15 INFO   Trader[binance]      logging_util.py:54       Created order on binance: {'info': {'symbol': 'BNBBUSD', 'orderId': 92649012, 'orderListId': -1, 'clientOrderId': 'Rs3iYgEwXOIKguodlZ41qj', 'transactTime': 1601640195000, 'price': '26.06730000', 'origQty': '0.44000000', 'executedQty': '0.00000000', 'cummulativeQuoteQty': '0.00000000', 'status': 'NEW', 'timeInForce': 'GTC', 'type': 'LIMIT', 'side': 'BUY'}, 'id': '92649012', 'timestamp': 1601640195000, 'datetime': '2020-10-02T12:03:15.000Z', 'lastTradeTimestamp': None, 'symbol': 'BNB/BUSD', 'type': 'limit', 'side': 'buy', 'price': 26.0673, 'amount': 0.44, 'cost': 0.0, 'average': None, 'filled': 0.0, 'remaining': 0.44, 'status': 'open', 'fee': None, 'trades': None}
2020-10-02 15:03:15 INFO   Trader[binance]      logging_util.py:54       Order creation : BNB/BUSD | BUY_LIMIT | Price : 26.0673 | Quantity : 0.44 | Status : OPEN 
2020-10-02 15:03:15 DEBUG  OrdersManager[binance] logging_util.py:50       Order added to open orders (total: 147 open orders)
2020-10-02 15:20:21 INFO   TaskManager          logging_util.py:54       Stopping threads ...
2020-10-02 15:20:21 DEBUG  Notification         logging_util.py:50       Twitter notification disabled
2020-10-02 15:20:21 DEBUG  Notification         logging_util.py:50       Telegram disabled
2020-10-02 15:20:21 INFO   RESTExchange[binance] logging_util.py:54       Closing connection.
2020-10-02 15:20:21 INFO   RESTExchange[binance] logging_util.py:54       Connection closed.
2020-10-02 15:20:21 INFO   TaskManager          logging_util.py:54       Threads stopped.
```
After I restart the bot, this is what it shows (we can see there are 143 loaded orders)
```
2020-10-02 15:20:23 INFO   OctoBot Launcher     start.py:127      Version : 0.3.9
2020-10-02 15:20:24 INFO   OctoBot Launcher     start.py:131      Loading config files...
2020-10-02 15:20:24 INFO   OctoBot Launcher     start.py:100      Disclaimer accepted by user.
2020-10-02 15:20:24 WARNING ServiceCreator       logging_util.py:58       RedditService can't be initialized: configuration is missing, wrong or incomplete !
2020-10-02 15:20:24 WARNING ServiceCreator       logging_util.py:58       TwitterService can't be initialized: configuration is missing, wrong or incomplete !
2020-10-02 15:20:24 INFO   WebService           logging_util.py:54       Interface successfully initialized and accessible at: http://127.0.1.1:5001.
2020-10-02 15:20:24 DEBUG  Notification         logging_util.py:50       Twitter notification disabled
2020-10-02 15:20:24 DEBUG  Notification         logging_util.py:50       Telegram disabled
2020-10-02 15:20:25 INFO   ExchangeDispatcher[binance] logging_util.py:54       online with REST api and web socket api
2020-10-02 15:20:25 INFO   Portfolio[binance]   logging_util.py:54       Current Portfolio : 
{.....some log showing all the loaded orders and the Binance portfolio....}
2020-10-02 15:20:26 INFO   Trader[binance]      logging_util.py:54       Order loaded : BNB/BUSD | BUY_LIMIT | Price : 25.9808 | Quantity : 0.46 | Status : OPEN 
2020-10-02 15:20:26 DEBUG  OrdersManager[binance] logging_util.py:50       Order added to open orders (total: 142 open orders)
2020-10-02 15:20:26 INFO   Trader[binance]      logging_util.py:54       Order loaded : BNB/BUSD | BUY_LIMIT | Price : 26.0673 | Quantity : 0.44 | Status : OPEN 
2020-10-02 15:20:26 DEBUG  OrdersManager[binance] logging_util.py:50       Order added to open orders (total: 143 open orders)
2020-10-02 15:20:26 DEBUG  Trader[binance]      logging_util.py:50       Enabled on binance
2020-10-02 15:20:26 INFO   TaskManager          logging_util.py:54       Evaluation tasks started...
```

**Comment by GuillaumeDSM** [comment]
Hi @AndreiMihalea,
There are sometimes issues with order notifications that might not be received by the binance websocket. This will be fixed in 0.4 (currently in beta) as we are completely changing the way orders are refreshed.

**Comment by AndreiMihalea** [comment]
Thank you for the clarification.
